accounts/admin.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from django.contrib.auth.forms import UserChangeForm, UserCreationForm
+from django import forms
+
+from accounts.models import User, Organization, Admin, Agent
+
+
+# Register your models here.
+# admin.site.register(User)
+admin.site.register(Organization)
+admin.site.register(Admin)
+admin.site.register(Agent)
+
+
+
+class AdminInline(admin.StackedInline):
+    model = Admin
+    can_delete = False
+    verbose_name_plural = 'Admin Profile'
+    fk_name = 'user'
+
+
+class CustomUserChangeForm(UserChangeForm):
+    class Meta(UserChangeForm.Meta):
+        model = User
+
+
+class CustomUserCreationForm(UserCreationForm):
+    organization = forms.ModelChoiceField(
+        queryset=Organization.objects.all(),
+        required=False,
+        help_text='Select an organization or create a new one.',
+    )
+
+    class Meta(UserCreationForm.Meta):
+        model = User
+
+    def save(self, commit=True):
+        user = super().save(commit=False)
+        user.is_staff = True
+        user.is_superuser = True
+
+        org = self.cleaned_data.get('organization')
+        if not org:
+            org = Organization.objects.create(name='Default Organization')
+
+        if commit:
+            user.save()
+            Admin.objects.create(user=user, org=org)
+
+        return user
```

Comment:
Why are these in admin.py?

Reply:
The forms are related to the modification of django admin, therefore I thought it would be fine to place them in the admin.py